src/app/shared/form/builder/ds-dynamic-form-ui/models/scrollable-dropdown/dynamic-scrollable-dropdown.component.ts

```diff
@@ -119,6 +119,7 @@ export class DsDynamicScrollableDropdownComponent extends DynamicFormControlComp
 
   onSelect(event) {
     this.group.markAsDirty();
+    console.log('onSelect event', event);
```

Comment:
Please remove this logging statement (or comment out)

Reply:
Added TypeDoc